packages/react-scripts/template/README.md

```diff
@@ -430,8 +429,8 @@ Then we can change `start` and `build` scripts to include the CSS preprocessor c
 
 ```diff
    "scripts": {
-     "build-css": "node-sass src/ -o src/",
-     "watch-css": "npm run build-css && node-sass src/ -o src/ --watch --recursive",
+     "build-css": "node-sass-chokidar src/ -o src/",
+     "watch-css": "npm run build-css && node-sass-chokidar src/ -o src/ --watch --recursive",
```

Comment:
like @kellyrmilligan said, custom scss loaders are not supported at the moment.

 - One option you have available is to `npm run eject` and add a scss loader to `config/webpack.config.*.js`. *then* you should be able to `import 'styles.scss';`
 - Another option is to output the css to a separate folder with `node-sass-chokidar` with the `-o <path>` option, however, this will make importing the css a huge pain if you use self-contained component hierarchy pattern like I do.

I don't think the devs on this see this as highest priority, but perhaps you can convince them to let you open a PR providing an entry point or env variable into the web-pack-config WITHOUT running eject?